src/bulk/common.ts

```diff
@@ -649,16 +655,17 @@ function handleMongoWriteConcernError(
 ) {
   mergeBatchResults(batch, bulkResult, undefined, err.result);
 
+  // TODO: make more reasonable
```

Comment:
This needs to be created as a ticket in the error improvements epic, currently captured in doc

Reply:
NODE-3337